src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthorizationBackend.java

```diff
@@ -257,9 +257,11 @@ private static Connection getConnection0(final Settings settings, final Path con
         Connection connection = null;
         Exception lastException = null;
 
+        final boolean isDebugEnabled = log.isDebugEnabled();
```

Comment:
can we make these class level fields, then we can utilize them for the subclasses as well